src/python/arcor2_arserver/rpc/objects.py

```diff
@@ -139,7 +134,7 @@ async def focus_object_done_cb(req: srpc.o.FocusObjectDone.Request, ui: WsClient
     if obj_id not in FOCUS_OBJECT:
         raise Arcor2Exception("focusObjectStart/focusObject has to be called first.")
 
-    await ensure_locked(req.args.id, "ui.owner")
+    await ensure_locked(req.args.id, glob.USERS.user_name(ui))
```

Comment:
Jenom detail, ale možná by to mohlo být `await ensure_locked(req.args.id, ui)` a to `glob.USERS.user_name(ui)` by bylo v tom `ensure_locked`? Malinko by se tím zjednodušil kód.

Reply:
Určitě dobrý nápad, předělám